test/e2e/manifest/manifest.go

```diff
@@ -117,30 +82,5 @@ func DaemonSetFromManifest(fileName, ns string) (*appsv1.DaemonSet, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = runtime.DecodeInto(scheme.Codecs.UniversalDecoder(), json, &ds)
-	if err != nil {
-		return nil, err
-	}
-	ds.Namespace = ns
-	return &ds, nil
-}
-
-// RoleFromManifest returns a Role from a manifest stored in fileName in the Namespace indicated by ns.
-func RoleFromManifest(fileName, ns string) (*rbacv1.Role, error) {
```

Comment:
was this function unused?

Reply:
yes . it is unused .

Reply:
Unused since the commit 38efff564c739d3e5d5aba90aca0498413a1907f